test/parser_test.rb

```diff
@@ -1,14 +1,15 @@
+# frozen_string_literal: true
 require_relative "helper"
 require 'kasket/query_parser'
 
 describe Kasket::QueryParser do
   def parse(options)
     scope = Post
-    options.each do |k,v|
+    options.each do |k, v|
       scope = case k
-      when :conditions then scope.where(v)
-      else
-        scope.send(k, v)
+              when :conditions then scope.where(v)
```

Comment:
Again, too much indentation for my taste. I’m sure there’s a configuration options for allowing 2-space indentation here.